CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.4.2
+
+This edge release contains a number of bug fixes.
+
+* CRDs
+  * Fixed Helm uninstall issue for HttpRoute
+
+* CNI
+  * Fixed incompatibility issue with AWS CNI addon in EKS, that was
+    forbidding pods to acquire networking after scaling up nodes.
+  * Added --set flag to install-cni plugin
+
+* Control Plane
+  * Add `trust_dns=error` to default proxy log level
```

Comment:
Can you also add the user-facing consequence? This silences warnings in the proxy logs (so this belongs under the proxy heading IMO) from that module, that are generally spurious.